<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>app_webseite</class>
 <widget class="QMainWindow" name="app_webseite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_heart">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>获取心跳数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_temp">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>获取体温数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sql">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>将数据缓存到本地</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tedit_sndquest">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbrow_rec">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sndquest">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
